packages/editor/src/components/parts/program-start/event/useStartData.ts

```diff
@@ -31,6 +31,8 @@ export function useStartData(): ConfigDataContext<ProgramStartData> & {
       produce(draft => {
         draft.javaClass = config.initConfig.javaClass;
         draft.permission = config.initConfig.permission;
+        draft.exceptionHandler = config.initConfig.exceptionHandler;
+        draft.timeout = config.initConfig.timeout;

```

Comment:
Why does this also reset the exceptionHandler and the timeout? Those fields are not supported by the ProgrammStartEvent right? At least those fields seems not to be on the UI...